src/components/Articles.jsx

```diff
@@ -1,11 +1,90 @@
 import React from "react";
+import { NavLink } from "react-router-dom";
+import Article from "./Article";
+import goback from "../images/goback.png";
+import { useEffect, useState } from "react";
+import { getArticles } from "../../api";
+import { getTopics } from "../../api";
+import "./Articles.css"
+
 
 function Articles() {
+  const [articles, setArticles] = useState([]);
+  const [search, setSearch] = useState("");
+  const [topics, setTopics] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((res) => {
+      const newList = res.filter((article) => article.title.toLowerCase().includes(search.toLowerCase()));
+      setArticles(newList);
+    });
+    getTopics().then((res) => {
```

Comment:
Topics are not dealt with until ticket 9. Please delete this and only add functionality that is needed for the specific ticket you are doing. Not only will it take you longer to do but will also me more difficult for us to review🙂